packages/cli/src/services/check-parser/parser.ts

```diff
@@ -133,9 +133,9 @@ export class Parser {
     }
   }
 
-  async getFilesAndDependencies (playwrightConfig: PlaywrightConfig):
+  async getFilesAndDependencies (playwrightConfig: PlaywrightConfig, playwrightConfigFilePath: string):
```

Comment:
Could you instead make `playwrightConfig` aware of its location? I already knows about the `dir`